Makefile

```diff
@@ -47,7 +47,12 @@ build-revision:
 	gox -verbose ${GO_LDFLAGS} -os="${BUILD_OS}" -arch="${BUILD_ARCH}" -output="${DIST_DIR}/${REVISION}/deis-${REVISION}-{{.OS}}-{{.Arch}}" .
 
 # This is supposed to be run within a docker container
-build-tag:
+build-stable:
+	$(eval GO_LDFLAGS = -ldflags '-X ${repo_path}/version.Version=${GIT_TAG}')
+	gox -verbose ${GO_LDFLAGS} -os="${BUILD_OS}" -arch="${BUILD_ARCH}" -output="${DIST_DIR}/deis-stable-{{.OS}}-{{.Arch}}" .
+
+# This is supposed to be run within a docker container
+build-tag: build-stable
```

Comment:
Are you sure you want to tie `build-tag` to `build-stable`? I could see cases for running them separately, for example, building the two in parallel.